pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -35,45 +32,38 @@ func NewConfigObserver(
 	operatorConfigInformers operatorv1informers.SharedInformerFactory,
 	configInformers configinformers.SharedInformerFactory,
 	eventRecorder events.Recorder,
-) *ConfigObserver {
-	c := &ConfigObserver{
-		ConfigObserver: configobserver.NewConfigObserver(
-			operatorClient,
-			eventRecorder,
-			configobservation.Listers{
-				ResourceSync:        resourceSyncer,
-				APIServerLister_:    configInformers.Config().V1().APIServers().Lister(),
-				ImageConfigLister:   configInformers.Config().V1().Images().Lister(),
-				ProjectConfigLister: configInformers.Config().V1().Projects().Lister(),
-				ProxyLister_:        configInformers.Config().V1().Proxies().Lister(),
-				IngressConfigLister: configInformers.Config().V1().Ingresses().Lister(),
-				EndpointsLister:     kubeInformersForEtcdNamespace.Core().V1().Endpoints().Lister(),
-				SecretLister_:       kubeInformers.Core().V1().Secrets().Lister(),
-				PreRunCachesSynced: []cache.InformerSynced{
-					operatorConfigInformers.Operator().V1().OpenShiftAPIServers().Informer().HasSynced,
-					configInformers.Config().V1().Images().Informer().HasSynced,
-					configInformers.Config().V1().Projects().Informer().HasSynced,
-					configInformers.Config().V1().Proxies().Informer().HasSynced,
-					configInformers.Config().V1().Ingresses().Informer().HasSynced,
-				},
+) factory.Controller {
+	c := configobserver.NewConfigObserver(
+		operatorClient,
+		eventRecorder,
+		configobservation.Listers{
```

Comment:
as I understand it now (after the recent changes in `library-go`) the reactors will be directly taken from the listers. I think only this one `operatorConfigInformers.Operator().V1().OpenShiftAPIServers().Informer()` is missing. 

There is no way to add it to `Lister` nor wire it. Does it mean we will not react on config updates (`spec.ObservedConfig.Raw`)?

Reply:
this produces the observed config. You mean that a user could tweak it and this controller won't undo the changes?

Reply:
Double checked: the target config controller watches the operator config. So we notice changes.

Reply:
> You mean that a user could tweak it and this controller won't undo the changes?

Yeah, since it won't react on config changes (but it did in the past)

Reply:
Is this okay then ?

Reply:
Feel free to fix it in follow-up